sdk/ml/azure-ai-ml/azure/ai/ml/_schema/monitoring/compute.py

```diff
@@ -0,0 +1,36 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+# pylint: disable=unused-argument
+
+from marshmallow import fields, post_load
+from azure.ai.ml._schema.core.schema import PatchedSchemaMeta
+from azure.ai.ml._schema.core.fields import StringTransformedEnum, NestedField
+from azure.ai.ml.constants._monitoring import COMPUTE_AML_TYPE
+
+
+class ComputeIdentitySchema(metaclass=PatchedSchemaMeta):
```

Comment:
as we dicussed last week, we don't need this since compute identity is always `AMLToken`